ch_pipeline/core/dataquery.py

```diff
@@ -808,7 +817,7 @@ def finder_from_spec(spec, node_spoof=None):
     return fi
 
 
-def files_from_spec(spec, node_spoof=None):
+def files_from_spec(spec, node_spoof=None, ntries=1):
```

Comment:
Could consider adding docstring entries for `node_spoof` and `ntries`, and also fix typo in docstring: `dspec` should be `spec`